Source/ARTLocalDevice.m

```diff
@@ -110,4 +110,23 @@ - (BOOL)isRegistered {
     return _identityTokenDetails != nil;
 }
 
+- (void)clearStorage {
```

Comment:
Do we have 100% sure that `self.storage` operations will be always synchronized?

Reply:
https://github.com/ably/ably-cocoa/pull/1253/commits/a88c282c0b88e58eb22adf7b272c49b6e649e264